solidity/.soliumrc.json

```diff
@@ -5,6 +5,7 @@
     ],
     "rules": {
         "security/no-block-members": "off",
-        "security/no-inline-assembly": "off"
+        "security/no-inline-assembly": "off",
+        "security/no-call-value": "off"
```

Comment:
We don't need this.

Reply:
you'd rather have the solium-disable comment?

Reply:
What I really want is a way to annotate a function as “I know this is going to do a `.call.value` and I'm completely okay with that”, and only in the call path of that one function is anyone allowed to do `.call.value`. I would like this compiler-enforced.

In the absence of such a guard, this at least draws extra attention to where we're doing `call.value`. But I wish we could annotate the whole function.

Basically I'm trying to use a linter to do a compiler's job and it's terrible. But I think it's marginally less terrible than not doing so.

Reply:
just saw https://github.com/keep-network/tbtc/pull/569#discussion_r405896462

Reply:
Makes sense to use linter annotation over the rule if it's just one instance